src/Motions/Document.ts

```diff
@@ -4,6 +4,7 @@ import {Motion} from './Motion';
 export class MotionDocument extends Motion {
 
     private line: number;
+    private percent: number|undefined;
```

Comment:
Since we may only assign either `line` or `percent`, we should make them both optional like this:

```ts
private line?: number;
private percent?: number;
```